tasks/jasmine-chromeapp.js

```diff
@@ -7,73 +7,169 @@ module.exports = function (grunt) {
     chalk = require('chalk'),
     path = require('path'),
     async = require('async'),
-    fs = require('fs-extra'),
+    fs = require('fs'),
     pkg = require('../package.json'),
-    chrome = require('node-chrome-runner');
+    chrome = require('node-chrome-runner'),
+    fileSyncCmp = require('file-sync-cmp');
 
-  function addFiles(files, to, tagFilter) {
-    var tags = '';
+  /* copyFiles is adapted from:
```

Comment:
module.exports = function(grunt) doesn't work (grunt is not defined except for top level required things, maybe grunt magic on its module registration), but `require('grunt')` does. Done.